.github/workflows/ci.yaml

```diff
@@ -0,0 +1,104 @@
+name: CI
+
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+    branches: [master]
+  schedule:
+  - cron: 0 4 * * *
+jobs:
+  unit_tests:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        py_version: ['3.12', '3.13']
+    steps:
+    - name: Checkout commit
+      uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+    - name: Install python
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.py_version }}
+    - name: Cache packages
+      uses: actions/cache@v4
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-py-${{ matrix.py_version }}-${{ hashFiles('pyproject.toml') }}
+    - name: Install poetry
+      run: pip install poetry==2.1.3
+    - name: Install dependencies
+      run: make setup
+    - name: Lint
+      env:
+        CI_LINT_RUN: 1
+      run: |
+        echo "::add-matcher::.github/actionlint-matcher.json"
+        make lint
+    - name: Run unit tests
+      run: make test_unit
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v5
+      with:
+        files: .coverage.xml
+        flags: py${{ matrix.py_version }}
+        name: codecov
+  integration_tests:
+    name: Run integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Install python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.13"
+      - name: Cache packages
+        uses: actions/cache@v4
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-py-3.13-${{ hashFiles('pyproject.toml') }}
+      - name: Install poetry
+        run: pip install poetry==2.1.3
+      - name: Install dependencies
+        run: make setup
+      - name: Run integration tests
+        run: make test_integration
+  deploy:
```

Comment:
Please add an explicit job for 'all passed' stage, like here: https://github.com/neuro-inc/apolo-events-client/blob/master/.github/workflows/ci.yaml#L51-L62

It makes the CI more uniform with others